lib-object-oriented/line.js

```diff
@@ -14,6 +15,19 @@ export default class Line {
       this.symbolCount === 1 )
   }
 
+  shouldSkip = ( index, currentToken ) => {
+    if (( this.tagType === tokenVars.HYPHEN ||
+        this.tagType === tokenVars.ASTERISK ) &&
+        index === 1 &&
+        currentToken.type === tokenVars.SPACE ) {
+      return true
+    } else if ( this.tagType === tokenVars.NUMB &&
```

Comment:
if the second if clause also returns `true`, then you can combine the first and second `if` clauses